### ISSUE TITLE
[apex] ApexDoc shouldn't report missing @description, because it's not part of specification

### ISSUE THREAD

**Issue by pkozuchowski** [pmd/pmd#6148]
**Affects PMD Version:**  Latest - PMD 7.17.0

**Rule:** ApexDoc
https://docs.pmd-code.org/latest/pmd_rules_apex_documentation.html#apexdoc

**Description:**
`@description` tag is not part of ApexDoc specification, so rule shouldn't report that it's missing.
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_doc_format.htm

> Main Description
The main description is the first block of text within an ApexDoc comment. It doesn’t have an explicit tag. It provides a concise summary of the documented element.

**Code Sample demonstrating the issue:**

```java
/**
 * Encapsulation for Trigger context variables.
 * This class is used as flag interface in methods to ensure they are run in trigger context,
 * but may also be used to mock trigger execution in unit tests.
 * It also contains helper method for checking field changes.
 */
public virtual class TriggerContext {
}
```

**Expected outcome:**

PMD reports a violation at line 11, but that's wrong. That's a false positive.
 7  4 (Low)      pmd:ApexDoc                TriggerContext.cls:11:16  Missing ApexDoc @description

**Running PMD through:** sf code-analyzer run 
https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/overview

**Comment by adangel** [comment]
Duplicate of #6189
